### PR TITLE
Fixed footer

### LINKED ISSUE #2449: Footer is not at the bottom of the page
**Actual Behaviour**

Still, footer is not at the bottom of the page.

**Expected Behaviour**

The footer should be at the bottom of the page.

**Screenshots of the issue**

Capture

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2449

Changes: Fixed footer at the bottom of the page

Demo Link: https://pr-2466-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]

### CODE REVIEW

**Comment by samagragupta** [comment]
Reload the page when changed from desktop view to mobile view.

**Comment by samagragupta** [comment]
@akshatnitd Made the changes

**Comment by samagragupta** [comment]
@harsh-jindal @akshatnitd Fixed that

**Comment by samagragupta** [comment]
@akshatnitd Hope this is fine.

File: src/App.js [hunk 1]
```diff
@@ -151,89 +152,87 @@ class App extends Component {
...
-            <div
-              style={
-                applyFooterStyle.includes(pathname)
-                  ? { marginTop: '230px' }
-                  : null
-              }
-            >
-              {renderFooter}
-            </div>
+            <Components>
```
Comment (akshatnitd) [inline src/App.js]: Rename it RootContainer.

Comment (samagragupta) [reply src/App.js]: This is already defined

File: src/components/cms/BotBuilder/BotWizard.js [hunk 2]
```diff
@@ -104,11 +104,11 @@ const StepperCol = styled(Col)`
 `;
 
 const ContainerCol = styled(Col)`
-  position: fixed;
-  margin-left: 65%;
   height: 88%;
   margin-top: 10px;
-
+  position: sticky;
```
Comment (akshatnitd) [inline src/components/cms/BotBuilder/BotWizard.js]: Why added this? Is it required?

Comment (samagragupta) [reply src/components/cms/BotBuilder/BotWizard.js]: To make the preview section fix to the screen. But when footer comes up it will scroll up.

Comment (samagragupta) [reply src/components/cms/BotBuilder/BotWizard.js]: With previous do you mean: https://github.com/fossasia/susi.ai/pull/2466#discussion_r301667655

File: src/App.js (hunk 1 above)
Comment (samagragupta) [reply src/App.js]: I think it is already fixed for mobile and tab views.

Comment (akshatnitd) [reply src/App.js]: Rename Components to RootContainer
RootContainer -> div

**Comment by akshatnitd** [comment]
@samagragupta I hope I have made it clear https://github.com/fossasia/susi.ai/pull/2466#discussion_r301760984

File: src/App.js (hunk 1 above)
Comment (samagragupta) [reply src/App.js]: @akshatnitd Done

File: src/components/Footer/Footer.react.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import styled from 'styled-components';
 const FooterContainer = styled.div`
   background: #f6f6f6;
   width: 100%;
-  position: absolute;
+  position: relative;
```
Comment (akshatnitd) [inline src/components/Footer/Footer.react.js]: Why do you need position now? Also, left and bottom?

Comment (samagragupta) [reply src/components/Footer/Footer.react.js]: Removed them

**Pull request merged**